src/preload/preload_interface.h

```diff
@@ -240,7 +246,7 @@ static inline const char* extract_file_name(const char* s) {
  * without bumping SYSCALLBUF_PROTOCOL_VERSION.
  */
 struct syscall_patch_hook {
-  uint8_t is_multi_instruction;
+  uint8_t flags;
   uint8_t next_instruction_length;
```

Comment:
Should this be renamed to `other_instruction_length` or something like that?

Reply:
Going with `patch_region_...`